scripts/ci-build-kind.sh

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env bash
```

Comment:
Probably we should drop the ci prefix of the script. The reason why the other scripts have it, is because they are used in CI (i.e. Prow). Usually they are directly called by the ProwJobs

Reply:
Can you please also drop the ci prefix from ci-install-mdbook.sh & ci-lint-dockerfiles.sh? (+ change usages in the CAPI repo, they are not used by test-infra/Prow, so nothing to do there)